src/artm/core/dictionary.h

```diff
@@ -46,7 +46,7 @@ class Dictionary {
   std::vector<DictionaryEntry> entries_;
   std::unordered_map<Token, int, TokenHasher> token_index_;
   std::unordered_map<int, Token> index_token_;
-  std::unordered_map<int, std::unordered_map<int, int> > cooc_values_;
+  std::unordered_map<int, std::unordered_map<int, std::vector<float> > > cooc_values_;
```

Comment:
No, not any e-mail, we discussed it during meeting. It'd be better to ask Anya about it.

07.07.2015, 00:48, "sashafrey" <notifications@github.com>:
> In src/artm/core/dictionary.h:
>
>> @@ -46,7 +46,7 @@ class Dictionary { > std::vector<DictionaryEntry> entries_; > std::unordered_map<Token, int, TokenHasher> token_index_; > std::unordered_map<int, Token> index_token_; > - std::unordered_map<int, std::unordered_map<int, int> > cooc_values_; > + std::unordered_map<int, std::unordered_map<int, std::vector<float> > > cooc_values_;
>
> Do you have that discussion somewhere in e-mail? I would like to see examples of what does not fit (1)
>
> —
> Reply to this email directly or view it on GitHub.